stories/topics/tech-note-aws-backup-restore.adoc

```diff
@@ -0,0 +1,23 @@
+[id="tech-note-aws-backup-restore"]
+
+= Backup and Restore
+
+. The `aws_rds_db_snapshot_arn` optional parameter which allowed customers to specify a specific AWS RDS Snapshot to use for restoring their deployment is no longer supported. The restore process no longer uses AWS RDS Snapshots to restore a deployment and instead relies on the backup files generated by the Ansible on Clouds Ops Container backup playbook. The Ansible on Clouds Ops Container restore playbook does not use any provided RDS snapshot passed as a parameter.
+
+. A restored {AAPonAWS} deployment does not maintain all secrets from the original {AAPonAWS} deployment. If using your own certificates, they must be reset after a restore.
+
++
+.Secrets that are maintained:
+
+* `<stack_name>-aap-admin-secret`
+* `<stack_name>-aap-secret-key`
+
++
```

Comment:
Delete line 15

Reply:
same with here I saw that removing the + removes the indentation